src/ddmd/codegen_ast_family.d

```diff
@@ -0,0 +1,18 @@
+module ddmd.codegen_ast_family;
+
+struct CodegenASTFamily
+{
+    import ddmd.aggregate;
```

Comment:
Good point. @RazvanN7 this is clever but let's create a unittest build with a separate AST family called e.g. `ASTNull` that has the ABSOLUTE MINIMAL types that make the parser build.

Then we need to parse a few files using `Parser!ASTNull` and place that in a unittest.